`make test_python` fails on ubuntu vm

https://app.circleci.com/pipelines/github/magma/magma/25777/workflows/ac376a02-53f7-43b5-8c27-37c49d91c265/jobs/274605
```
[magma@10.240.0.53] out: [vagrant@127.0.0.1:2222] out: See system logs and 'systemctl status magma@dummy_service.service' for details.
[magma@10.240.0.53] out: [vagrant@127.0.0.1:2222] out: ..........................................F
[magma@10.240.0.53] out: [vagrant@127.0.0.1:2222] out: ======================================================================
[magma@10.240.0.53] out: [vagrant@127.0.0.1:2222] out: FAIL: test_pylint (tests.pylint_tests.MagmaPyLintTest)
[magma@10.240.0.53] out: [vagrant@127.0.0.1:2222] out: ----------------------------------------------------------------------
[magma@10.240.0.53] out: [vagrant@127.0.0.1:2222] out: Traceback (most recent call last):
[magma@10.240.0.53] out: [vagrant@127.0.0.1:2222] out:   File "/home/vagrant/magma/orc8r/gateway/python/magma/tests/pylint_tests.py", line 57, in test_pylint
[magma@10.240.0.53] out: [vagrant@127.0.0.1:2222] out:     py_wrap.assertNoLintErrors(path)
[magma@10.240.0.53] out: [vagrant@127.0.0.1:2222] out:   File "/home/vagrant/magma/lte/gateway/python/magma/tests/pylint_wrapper.py", line 97, in assertNoLintErrors
[magma@10.240.0.53] out: [vagrant@127.0.0.1:2222] out:     raise AssertionError(msg)
[magma@10.240.0.53] out: [vagrant@127.0.0.1:2222] out: AssertionError: PyLint found errors:
[magma@10.240.0.53] out: [vagrant@127.0.0.1:2222] out: ************* Module magmad.metrics
[magma@10.240.0.53] out: [vagrant@127.0.0.1:2222] out: W1511: 245, 19: Using deprecated argument loop of method sleep() (deprecated-argument)
[magma@10.240.0.53] out: [vagrant@127.0.0.1:2222] out: ************* Module magmad.check.subprocess_workflow
[magma@10.240.0.53] out: [vagrant@127.0.0.1:2222] out: W1511: 75, 8: Using deprecated argument loop of method create_subprocess_exec() (deprecated-argument)
[magma@10.240.0.53] out: [vagrant@127.0.0.1:2222] out: W1511: 83, 25: Using deprecated argument loop of method gather() (deprecated-argument)
[magma@10.240.0.53] out: [vagrant@127.0.0.1:2222] out: ************* Module magmad.upgrade.upgrader
[magma@10.240.0.53] out: [vagrant@127.0.0.1:2222] out: W1511: 80, 15: Using deprecated argument loop of method sleep() (deprecated-argument)
[magma@10.240.0.53] out: [vagrant@127.0.0.1:2222] out: W1511: 95, 19: Using deprecated argument loop of method sleep() (deprecated-argument)
[magma@10.240.0.53] out: [vagrant@127.0.0.1:2222] out: 
[magma@10.240.0.53] out: [vagrant@127.0.0.1:2222] out: 
```

--- a/orc8r/gateway/python/magma/magmad/check/subprocess_workflow.py
+++ b/orc8r/gateway/python/magma/magmad/check/subprocess_workflow.py
@@ -76,13 +76,11 @@ def exec_and_parse_subprocesses_async(
             *arg_list_func(param),
             stdout=asyncio.subprocess.PIPE,
             stderr=asyncio.subprocess.PIPE,
-            loop=loop,
         ) for param in params
     ]
     subprocs = yield from asyncio.gather(*futures)
     outputs = yield from asyncio.gather(
         *[subproc.communicate() for subproc in subprocs],
-        loop=loop,
     )
     return _parse_results(params, outputs, result_parser_func)
 

--- a/orc8r/gateway/python/magma/magmad/check/tests/subprocess_workflow_tests.py
+++ b/orc8r/gateway/python/magma/magmad/check/tests/subprocess_workflow_tests.py
@@ -102,13 +102,11 @@ class SubprocessWorkflowTests(unittest.TestCase):
                     'param1', 'a', 'b',
                     stdout=subprocess.PIPE,
                     stderr=subprocess.PIPE,
-                    loop=loop,
                 ),
                 mock.call(
                     'param2', 'a', 'b',
                     stdout=subprocess.PIPE,
                     stderr=subprocess.PIPE,
-                    loop=loop,
                 ),
             ])
             self.mock_parser_callback.assert_has_calls([
@@ -164,13 +162,11 @@ class SubprocessWorkflowTests(unittest.TestCase):
                     'param1', 'a', 'b',
                     stdout=subprocess.PIPE,
                     stderr=subprocess.PIPE,
-                    loop=loop,
                 ),
                 mock.call(
                     'param2', 'a', 'b',
                     stdout=subprocess.PIPE,
                     stderr=subprocess.PIPE,
-                    loop=loop,
                 ),
             ])
 

--- a/orc8r/gateway/python/magma/magmad/main.py
+++ b/orc8r/gateway/python/magma/magmad/main.py
@@ -237,7 +237,7 @@ def main():
         command_executor = get_command_executor_impl(service)
 
     # Start loop to monitor unattended upgrade status
-    service.loop.create_task(monitor_unattended_upgrade_status(service.loop))
+    service.loop.create_task(monitor_unattended_upgrade_status())
 
     # Add all servicers to the server
     magmad_servicer = MagmadRpcServicer(

--- a/orc8r/gateway/python/magma/magmad/metrics.py
+++ b/orc8r/gateway/python/magma/magmad/metrics.py
@@ -225,7 +225,7 @@ def _collect_ping_metrics(ping_params, loop=None):
 
 
 @asyncio.coroutine
-def monitor_unattended_upgrade_status(loop):
+def monitor_unattended_upgrade_status():
     """
     Call to poll the unattended upgrade status and set the corresponding metric
     """
@@ -242,4 +242,4 @@ def monitor_unattended_upgrade_status(loop):
                         break
         logging.debug('Unattended upgrade status is %d', status)
         UNATTENDED_UPGRADE_STATUS.set(status)
-        yield from asyncio.sleep(POLL_INTERVAL_SECONDS, loop=loop)
+        yield from asyncio.sleep(POLL_INTERVAL_SECONDS)

--- a/orc8r/gateway/python/magma/magmad/upgrade/upgrader.py
+++ b/orc8r/gateway/python/magma/magmad/upgrade/upgrader.py
@@ -77,7 +77,7 @@ def start_upgrade_loop(magmad_service, upgrader):
     # even the first checkin. Delay a little bit so the device can
     # record stats/checkin/give someone an opportunity to disable
     logging.info("Waiting before checking for updates for the first time...")
-    yield from asyncio.sleep(120, loop=magmad_service.loop)
+    yield from asyncio.sleep(120)
 
     while True:
         logging.info('Checking for upgrade...')
@@ -92,7 +92,7 @@ def start_upgrade_loop(magmad_service, upgrader):
             60,
             magmad_service.mconfig.autoupgrade_poll_interval,
         )
-        yield from asyncio.sleep(poll_interval, loop=magmad_service.loop)
+        yield from asyncio.sleep(poll_interval)
 
 
 def _get_target_version(magmad_mconfig):

